rct_optimizations/test/extrinsic_hand_eye_utest.cpp

```diff
@@ -226,6 +254,66 @@ TYPED_TEST(HandEyeTest, RandomAroundAnswerInitialConditions)
   EXPECT_LT(count, max_attempts);
 }
 
+TYPED_TEST(HandEyeTest, RandomAroundAnswerInitialConditions_MoreHemispheres)
+{
+  const std::size_t n = 10;
+  const std::size_t max_attempts = 2 * n;
+  std::size_t count = 0;
+
+
+  Eigen::Isometry3d offset = Eigen::Isometry3d::Identity();
+  offset.translation().x() = 0.1;
+
+  std::vector<std::shared_ptr<test::PoseGenerator>> pgs;
+  pgs.push_back(std::make_shared<test::RandomZRotHemispherePoseGenerator>(2.0, 10, 10, Eigen::Isometry3d::Identity()));
+  pgs.push_back(std::make_shared<test::RandomZRotHemispherePoseGenerator>(2.0, 10, 10, offset));
```

Comment:
Should we add more radii (shells) to the hemisphere also?

Reply:
This is one of those tests that gets run 10 times, and it tends to chug if lots of hemispheres are added. I'll change so it just runs once with a larger number of observations.